docs/glsl-fuzz-develop.md

```diff
@@ -158,7 +201,15 @@ cd gles-worker/
 
 ## Vulkan worker developer documentation
```

Comment:
Should we call this "Legacy Vulkan worker developer documentation"?

Reply:
Done.